kubernetes/kserve/build_image.sh

```diff
@@ -23,14 +28,32 @@ do
           DOCKER_FILE="Dockerfile.dev"
           shift
           ;;
+        -n|--nightly)
+          BUILD_NIGHTLY=true
+          shift
+          ;;
         -t|--tag)
-          DOCKER_TAG="$2"
+          CUSTOM_TAG="$2"
+          USE_CUSTOM_TAG=true
```

Comment:
Please add documentation steps for usage for this. eg what will be the actual command line to generate the GPU kserve image?

Reply:
This change does not affect the cli parameters and will not require doc change.